projects/anyhedge/index.js

```diff
@@ -47,20 +47,22 @@ async function tvl({timestamp}) {
 }
 
 module.exports = {
-  methodology: "Scrape the blockchain and filter for spent transaction outputs that match the contract's input script template. Aggregate them to compute TVL. The TVL data lags by contract duration since contracts are secret until settled. So, TVL at the current time will always be 0 and can only be calculated in retrospect and stats back-filled when contracts are revealed. For this reason, the code cuts-off the data at 31 days ago. See here for more details: https://gitlab.com/0353F40E/anyhedge-stats/-/blob/master/readme.md",
+  methodology: "Scrape the blockchain and filter for spent transaction outputs that match the contract's input script template. Aggregate them to compute TVL. The TVL data lags by contract duration since contracts are secret until settled. So, TVL at the current time will always be 0 and can only be calculated in retrospect and stats back-filled when contracts are revealed. For this reason, the code cuts-off the data at 91 days ago. See here for more details: https://gitlab.com/0353F40E/anyhedge-stats/-/blob/master/readme.md",
   start: 1654787405,
   bitcoincash: { tvl },
   hallmarks: [
```

Comment:
hmm, somehow I missed this part before, can you cut down the # hallmarks to 3-4

If we have too many, it pollutes the charts and users end up disabling it everywhere

Reply:
@g1nt0ki done, and after you merge could you please purge old data and run a full backfill

due to max. duration change (30d to 90d) the current data is incomplete because it was picking up stats before all contracts would reveal themselves

the graph should match the one [here](https://gitlab.com/0353F40E/anyhedge-stats/-/blob/master/plots/plots.md) but display cut at 3 months before current date.